agent/util-scripts/pbench-tool-meister-start

```diff
@@ -924,7 +927,9 @@ def main(_prog: str, cli_params: Namespace) -> int:
         tm = dict(
             benchmark_run_dir=str(benchmark_run_dir),
             channel_prefix=cli_tm_channel_prefix,
-            tds_hostname=params["origin_host"] if "origin_host" in params else tool_data_sink.host,
+            tds_hostname=params["origin_host"]
+            if "origin_host" in params
+            else tool_data_sink.host,
```

Comment:
Oh, that is *so* ugly. I've come to really hate Python ternary logic just because `black` makes them so darned ugly. Almost like it's passive-aggressively trying to make us stop using them.

Reply:
This is a fall-out from the `black` update.  Basically, this module is not picked up by `black .`, but is by the pre-commit check.